Sources/TuistKit/Services/FocusService.swift

```diff
@@ -46,6 +50,10 @@ final class FocusService {
         if !noOpen {
             try opener.open(path: workspacePath)
         }
+        let time = String(format: "%.3f", timer.stop())
+
+        logger.notice("Project generated.", metadata: .success)
```

Comment:
Having the same output logic in two places might lead to inconsistencies in the future if we update it in one side and forget the other. I recommend extracting it into some place that can be reused across the focus and generate services.

Reply:
Just to clarify, what would you extract exactly? We have the `Clock` object, would you have a wrap that with those two `logger` lines? Or would you extract the string constants only?
I kind of feel that having Clock + `Total time taken` may make sense, but with `Project generated.` I have different feeling. Currently it may be a wrong text for either (`tuist generate` and `tuist focus`) commands as we may generate more than one project and a workspace as well etc... Maybe, it can be something more generic? Like `Focus completed.` vs `Generation completed.` for `tuist generate`?
WDYT? If you feel that the current text make sense for both I am happy to extract including that sentences as well :)

Reply:
I'm referring to the formatting and the content of the total time taken message. I'll leave it up to you to decide whether it makes sense to have a templated string or a class/method elsewhere that can easily be reused.
Sharing the motivation with you, these are common sources of inconsistency and they quickly add up to worsening the experience developers get using the CLI.

Reply:
Done @pepicrft, if it now looks good feel free to merge it. Thanks again for the review! 💐